webapp/src/utils/time.ts

```diff
@@ -39,3 +39,14 @@ export const formatYYYYMMddHHMMSS = (timestamp: number): string => {
   const seconds = String(date.getSeconds()).padStart(2, '0');
   return `${year}-${month}-${day} ${hours}:${minutes}:${seconds}`;
 }
+
+export const zonedDateFormatYYYYMMddHHMMSS = (utcDateTime: string): string => {
```

Comment:
Validate that the zone offset is 0

Reply:
My time zone is UTC+8. I have tested it.